### PR TITLE
VAGOV-4622: Force health service taxonomy references to use child terms.

### PR DESCRIPTION
**Pull request by kevwalsh**
To QA:
* Go to /node/add/regional_health_care_service_des
* In the Service name and description, note that the options are sorted alphabetically, and that you can only select child taxonomy terms. The parents listed at /admin/structure/taxonomy/manage/health_care_service_taxonomy/overview do not show (unless they are also child terms, like Primary Care and Mental Health)

### CODE REVIEW

**Comment by andyhawks** [comment]
rebuild

**Comment by ElijahLynn** [comment]
rebuild

**Comment by ElijahLynn** [comment]
failed due to container failing to build

**Comment by ElijahLynn** [comment]
rebuild

File: config/sync/field.field.node.regional_health_care_service_des.field_local_health_care_service_.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ id: node.regional_health_care_service_des.field_local_health_care_service_
 field_name: field_local_health_care_service_
 entity_type: node
 bundle: regional_health_care_service_des
-label: 'Local offering for this service'
+label: 'Facility-specific descriptions of this service'
 description: 'This will get automatically updated when you <a href="/node/add/health_care_local_health_service">create a local health service offering. '
```
Comment (swirtSJW) [inline config/sync/field.field.node.regional_health_care_service_des.field_local_health_care_service_.yml]: Space at end of sentence causes testing issues according to Daniel.

Comment (swirtSJW) [reply config/sync/field.field.node.regional_health_care_service_des.field_local_health_care_service_.yml]: I removed the space.

**Pull request merged**